README.md

```diff
@@ -52,14 +52,15 @@ or download the [starter project](https://github.com/mcollovati/quarkus-hilla-st
 <dependency>
     <groupId>com.github.mcollovati</groupId>
     <artifactId>quarkus-hilla</artifactId>
-    <version>2.2.1</version>
+    <version>2.3.0</version>
 </dependency>
 ```
 
 ## Compatibility Matrix
 
 |                                                                                                     Quarkus-Hilla                                                                                                      |                                                                               Quarkus                                                                               |                                                        Hilla                                                         |                                                                             Vaadin                                                                             |
 |:----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------:|:-------------------------------------------------------------------------------------------------------------------------------------------------------------------:|:--------------------------------------------------------------------------------------------------------------------:|:--------------------------------------------------------------------------------------------------------------------------------------------------------------:|
+| <picture><img alt="Maven Central 2.3" src="https://img.shields.io/maven-central/v/com.github.mcollovati/quarkus-hilla?style=for-the-badge&logo=apache-maven&versionPrefix=2.3" style="visibility: visible;"></picture> |  <picture><img alt="Quarkus 3.1+" src="https://img.shields.io/badge/QUARKUS-v3.1%2B-blue?style=for-the-badge&logo=Quarkus" style="visibility: visible;"></picture>  | <picture><img alt="Hilla 2.3" src="https://tinyurl.com/quarkus-hilla-h-2-3" style="visibility: visible;"></picture>  |   <picture><img alt="Flow 24.2" src="https://img.shields.io/badge/VAADIN-v24.2-blue?style=for-the-badge&logo=Vaadin" style="visibility: visible;"></picture>   |
```

Comment:
Must the vaadin version icon also increased to 24.3 or is it right to be 24.2 compatibly?

Reply:
No, Hilla 2.3 is compatible with Vaadin 24.2

Reply:
Then we should introduce the highlighting I mentioned